packages/harmony/src/components/layout/Box/Box.tsx

```diff
@@ -60,7 +60,7 @@ export const Box = styled.div<BoxProps>(
       paddingLeft: (padL && spacing[padL as SpacingOptions]) ?? padL,
       paddingRight: (padR && spacing[padR as SpacingOptions]) ?? padR,
       paddingBottom: (padB && spacing[padB as SpacingOptions]) ?? padB,
-      marginTop: marginT && spacing[marginT],
+      marginTop: (marginT && spacing[marginT as SpacingOptions]) ?? marginT,
```

Comment:
this a one-off thing? should this be done for the other margin props too?

Reply:
Yeah actually going to remove this since we don't need it, though we should probably normalize on this at some point